src/screenControl.py

```diff
@@ -105,9 +106,9 @@ def outputSide(self):
         if self.mode == COMMAND_MODE:
             usageLines = processInput.USAGE_COMMAND.split('\n')
         for index, usageLine in enumerate(usageLines):
-            self.stdscr.addstr(self.getMinY() + index, borderX + 2, usageLine)
+            self.stdscr.addstr(self.getMinY() + index, borderX + 2, usageLine, curses.color_pair(0))
```

Comment:
the `curses.color_pair(0)` here is a lot of copy-pasta, could we consolidate that somehow? maybe a `self.addStrStandard` or something that uses the default formatting?